src/PrestaShopBundle/Form/Admin/Improve/Design/Pages/CmsPageCategoryType.php

```diff
@@ -73,8 +74,12 @@ public function __construct(array $allCmsCategories, $isShopFeatureEnabled)
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $invalidCharactersForCatalogLabel = $this->trans('Invalid characters:', 'Admin.Notifications.Info') .  '<>;=#{}';
```

Comment:
```suggestion
        $invalidCharactersForCatalogLabel = $this->trans('Invalid characters:', 'Admin..Global') .  '<>;=#{}';
```

Reply:
Hi @Julievrz ,
There is an extra dot in: `Admin..Global`

Reply:
Oops, thank you @Arman-Hosseini !